### ISSUE TITLE
warning: `Application.get_env/2` discouraged use `Application.compile_env/3` instead

### ISSUE THREAD

**Issue by nelsonic** [dwyl/auth_plug#82]
Getting the following error in a downstream project that is using `auth_plug`: https://github.com/dwyl/phoenix-chat-example/pull/120
```
warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead
  lib/auth_plug.ex:22: AuthPlug
```

Should be a easy/quick fix. 💭

**Comment by nelsonic** [comment]
Still get the following compilation warnings:

```sh
Compiling 41 files (.ex)
warning: Application.get_env/3 is discouraged in the module body, use Application.compile_env/3 instead
  lib/plug/conn/status.ex:6: Plug.Conn.Status

warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead
  lib/plug/mime.ex:4: Plug.MIME

warning: System.stacktrace/0 is deprecated. Use __STACKTRACE__ instead
  lib/plug/conn/wrapper_error.ex:26: Plug.Conn.WrapperError.reraise/3
```

But at least these are all in the `plug` library and not `auth_plug`. 👌

**Comment by nelsonic** [comment]
Package published to https://hex.pm/packages/auth_plug/1.4.21 📦 

Change included in PR: #77
